package.json

```diff
@@ -54,6 +54,7 @@
     "prop-types": "^15.5.10",
     "react-native-animatable": "^1.1.0",
     "react-native-color": "0.0.10",
+    "react-native-haptic-feedback": "^1.11.0",
```

Comment:
The whole idea of the optional dependencies is that we don't force to install those dependencies for the user. 
This is to avoid huge bundle size with packages our consumer don't necessarily use/need. 

If the user decide to use a component that has a dependency on one of the optionals deps, they can install it.